projects/driver_controls/inc/input_event.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+// Define common typedefs for the various FSMs to use.
+
+#include "event_queue.h"
+
+// ID definitions for the driver input events.
+typedef enum {
+  INPUT_EVENT_POWER = 0,
+  INPUT_EVENT_PEDAL_BRAKE,
+  INPUT_EVENT_PEDAL_COAST,
+  INPUT_EVENT_PEDAL_PRESSED,
+  INPUT_EVENT_CRUISE_CONTROL,
+  INPUT_EVENT_CRUISE_CONTROL_INC,
+  INPUT_EVENT_CRUISE_CONTROL_DEC,
+  INPUT_EVENT_DIRECTION_SELECTOR_NEUTRAL,
+  INPUT_EVENT_DIRECTION_SELECTOR_DRIVE,
+  INPUT_EVENT_DIRECTION_SELECTOR_REVERSE,
+  INPUT_EVENT_TURN_SIGNAL_NONE,
+  INPUT_EVENT_TURN_SIGNAL_LEFT,
+  INPUT_EVENT_TURN_SIGNAL_RIGHT,
+  INPUT_EVENT_HAZARD_LIGHT,
+  INPUT_EVENT_MECHANICAL_BRAKE,
```

Comment:
I think it's nice to have idempotent events (i.e. they can be raised multiple times without issue). Not something that's super necessary, but it's one of those things to think about.